runtime/src/serde_snapshot.rs

```diff
@@ -242,7 +239,7 @@ where
         }};
     }
     match serde_style {
-        SerdeStyle::Newer => INTO!(TypeContextFuture),
+        SerdeStyle::Newer => INTO!(newer),
```

Comment:
When I add a new case here, the `INTO!` macro was complaining about using an `ident` that had scoping (i.e. `newer::Context` and `older::Context`). So the simple fix is to have the macro take the serde style, since the `Context` part is fixed.